Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Dockerfile for automated build of signal-cli
+#
+# Refer to the signal-cli github pages for detailed Usage.
+#
+FROM gradle:3.5-jdk7-alpine
+
+LABEL maintainer "Kayvan Sylvan <kayvansylvan@gmail.com>"
+
+USER root
```

Comment:
I hear you. I couldn't figure out an easy way to do that, and given that the install happens inside a `gradle:jdk7-alpine` container and does not touch the host on which you are running, I don't think it matters.

The alpine container is a minimal `busybox` based system. What to you propose as the way to run the build as a non-root user?